src/legacy/ui/public/vislib/visualizations/point_series/column_chart.js

```diff
@@ -27,7 +27,7 @@ const defaults = {
   showTooltip: true,
   color: undefined,
   fillColor: undefined,
-  showLabel: true,
+  showLabel: true
```

Comment:
I don't think there was a specific need to remove that comma? :-) Since we're linting in newer files to keeping the trailing comma (for better git compatibility), could you please leave the comma here.